### PR TITLE
Remove trailing whitespaces

### PR DESCRIPTION
**Pull request by pavel-tkachenko**

### CODE REVIEW

File: src/plugins/json/qjsonparser/json.cpp [hunk 1]
```diff
@@ -21,7 +21,7 @@
   well-formed JSON into a QVariant.
 
   The parser converts JSON types into QVariant types. For example, JSON
-  arrays are translated into QVariantList and JSON objects are translated
```
Comment (bjorn) [inline src/plugins/json/qjsonparser/json.cpp]: Please don't touch this file, it's 3rd party source code.

File: src/plugins/json/qjsonparser/jsonparser.cpp [hunk 2]
```diff
@@ -68,19 +68,19 @@ class JsonGrammar
 
 
 const char *const JsonGrammar::spell [] = {
-  "end of file", "string", "number", "{", "}", "[", "]", ":", ",", "false", 
-  "true", "null", "error",
```
Comment (bjorn) [inline src/plugins/json/qjsonparser/jsonparser.cpp]: Please don't touch this file either.

File: src/plugins/tbin/tbin/Map.hpp [hunk 3]
```diff
@@ -42,10 +42,10 @@ namespace tbin
         public:
             bool loadFromFile( const std::string& path );
             bool loadFromStream( std::istream& in );
-
```
Comment (bjorn) [inline src/plugins/tbin/tbin/Map.hpp]: Please don't change any files in the `tbin/tbin` folder, they're 3rd party code.

File: src/qtpropertybrowser/src/qtpropertybrowser.qrc [hunk 4]
```diff
@@ -20,4 +20,4 @@
         <file>images/cursor-whatsthis.png</file>
     </qresource>
 </RCC>
-
```
Comment (bjorn) [inline src/qtpropertybrowser/src/qtpropertybrowser.qrc]: Please don't change any files in `src/qtpropertybrowser` and `src/qtsingleapplication`. They are from the Qt Solutions repository and changes should be kept minimal in order to make synchronizing the code easier.

**Comment by pavel-tkachenko** [comment]
Done.

**Comment by bjorn** [comment]
Thanks! :-)

**Pull request merged**